index.d.ts

```diff
@@ -0,0 +1,17 @@
+import { NamedNode } from 'rdf-js';
+
+interface NamespaceBuilder {
+  (property: TemplateStringsArray): NamedNode;
+  (property: string): NamedNode;
+  [property: string]: NamedNode;
+}
+
+interface Factory {
+  namedNode(value: string): NamedNode;
+}
+
+export interface BuilderOptions {
+  factory?: Factory;
+}
+
+export default function namespace(baseIRI: string, options?: BuilderOptions): NamespaceBuilder;
```

Comment:
This is a CommonJS package. I think the export should be something like below.

```suggestion
-declare function namespace(baseIRI: string, options?: BuilderOptions): NamespaceBuilder;
+export = namespace
```

Reply:
I guess it's safe not needed to export `BuilderOptions`.